client/src/actions/index.js

```diff
@@ -265,5 +283,6 @@ export default {
   requestSingleGeneExpressionCountsForColoringPOST,
   requestGeneExpressionCountsPOST,
```

Comment:
`requestGeneExpressionCountsPOST` is no longer used outside this module (ie, it is now a module-internal helper).  It probably should not be exported.

Reply:
digging in more - it looks to me like `requestGeneExpressionCountsPOST()` can be delete, as it is a NOOP wrapper around `_doRequestExpressionData()`.   While `requestGeneExpressionCountsPOST()` does emit some actions, they are not used, and are identical in semantics to those emitted by `_doRequestExpressionData()`

Suggest removing this extraneous layer.